src/main/resources/assets/extended_industrialization/lang/ru_ru.json

```
@@ -0,0 +1,322 @@
+  "text.extended_industrialization.nano_suit_help_night_vision": "- Нажмите %s в надетом состоянии или %s при наведении, чтобы переключить Ночное зрение.",
+  "text.extended_industrialization.nano_suit_help_speed": "- Нажмите %s в надетом состоянии или %s при наведении, чтобы переключить Ускорение.",
+  "text.extended_industrialization.nano_suit_help_step": "- Нажмите %s в надетом состоянии или %s при наведении, чтобы переключить Повышение шага.",
+  "text.extended_industrialization.nano_suit_night_vision": "Ночное зрение: %s",
+  "text.extended_industrialization.nano_suit_night_vision_toggled_off": "Ночное зрение отключено",
+  "text.extended_industrialization.nano_suit_night_vision_toggled_on": "Ночное зрение включено",
+  "text.extended_industrialization.nano_suit_speed": "Скорость: %s",
+  "text.extended_industrialization.nano_suit_speed_toggled_off": "Ускорение отключено",
+  "text.extended_industrialization.nano_suit_speed_toggled_on": "Ускорение включено",
+  "text.extended_industrialization.nano_suit_step": "Шаг: %s",
+  "text.extended_industrialization.nano_suit_step_toggled_off": "Повышение шага отключено",
+  "text.extended_industrialization.nano_suit_step_toggled_on": "Повышение шага включено",
+  "text.extended_industrialization.photovoltaic_cell_eu": "Будет производить до %s при установке в солнечную панель.",
+  "text.extended_industrialization.photovoltaic_cell_remaining_operation_time": "Оставшееся время работы: %s",
+  "text.extended_industrialization.photovoltaic_cell_remaining_operation_time_minutes": "Оставшееся время работы: %s мин.",
+  "text.extended_industrialization.priority": "Приоритет: ",
+  "text.extended_industrialization.processing_array_batch_size": "Размер партии определяется количеством подключённых машин.",
+  "text.extended_industrialization.processing_array_eu_cost_multiplier": "Работает при затратах EU в %s.",
+  "text.extended_industrialization.processing_array_machine_input": "Вставьте электрические машины для параллельной работы.",
+  "text.extended_industrialization.processing_array_recipe": "Может пакетно выполнять рецепты любой одноблочной электрической машины, подключённой к нему.",
+  "text.extended_industrialization.processing_array_size": "Машин: %d",
+  "text.extended_industrialization.rainbow": "Радуга",
+  "text.extended_industrialization.robot_auto_feeder_help_1": "Автоматически берёт Консервы из вашего инвентаря и кормит вас.",
+  "text.extended_industrialization.robot_auto_feeder_help_2": "Работает с предметами-контейнерами, такими как рюкзаки.",
+  "text.extended_industrialization.solar_boiler_calcification": "Со временем будет покрываться накипью и терять эффективность до минимума %s без использования %s. Удар топором по котлу сбросит накипь.",
+  "text.extended_industrialization.solar_efficiency": "Солнечная эффективность: %d %％",
```

Что случилось с третьим знаком %?
```suggestion
  "text.extended_industrialization.solar_efficiency": "Солнечная эффективность: %d %%",
```